pkg/kubelet/checkpointmanager/checkpoint_manager_test.go

```diff
@@ -0,0 +1,170 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package checkpointmanager
+
+import (
+	"encoding/json"
+	"hash/fnv"
+	"sort"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"k8s.io/kubernetes/pkg/kubelet/checkpointmanager/errors"
+	utilstore "k8s.io/kubernetes/pkg/kubelet/checkpointmanager/testing"
+	hashutil "k8s.io/kubernetes/pkg/util/hash"
+)
+
+type Protocol string
```

Comment:
Can you add a comment that this struct is used as an example of a struct that can be checkpointed.  We should also not export these structs if that is what is used for.

Reply:
done.